travis build failed: cannot open libgfortran.so.1

Refs: [travis log](https://travis-ci.org/ornlneutronimaging/iVenus/jobs/107233269#L562)

Error msg:
>   from ._ufuncs import *
E   ImportError: libgfortran.so.1: cannot open shared object file: No such file or directory

--- a/.travis.yml
+++ b/.travis.yml
@@ -28,7 +28,7 @@ install:
   - conda config --set always_yes true
   - conda update conda
   - conda create -n testenv pip pytest python=$TRAVIS_PYTHON_VERSION
-  - conda install -n testenv pyyaml numpy scipy matplotlib astropy 
+  - conda install -n testenv pyyaml numpy scipy=0.16.0 matplotlib astropy 
 #  - conda install -n testenv -c scikit-xray xraylib lmfit=0.8.3 netcdf4 # install from scikit-xray channel
   - source activate testenv
   - pip install progressbar2

